utils/distarray_conda-env.yml

```diff
@@ -0,0 +1,44 @@
+name: distarray
+dependencies:
```

Comment:
Can these be narrowed down to the top-level dependencies (and let conda figure out the rest)?